packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -126,6 +126,13 @@ abstract class LambdaAuthorizer extends Authorizer implements IAuthorizer {
       },
     });
   }
+
+  /**
+   * constructs the authorizerURIArn.
+   */
+  protected authorizerURIArn() {
+    return `arn:${Stack.of(this).partition}:apigateway:${Stack.of(this).region}:lambda:path/2015-03-31/functions/${this.handler.functionArn}/invocations`;
+  }
```

Comment:
This will expose this API as part of a public API. That is not necessary here.

Instead, make this a standalone function at the bottom of the file. 

```ts
  function lambdaAuthorizerArn(handler: Function) {
    return `arn:${Stack.of(handler).partition}:apigateway:${Stack.of(handler).region}:lambda:path/2015-03-31/functions/${handler.functionArn}/invocations`;
  }
```